CONTRIBUTING.md

```diff
@@ -118,6 +118,12 @@ NB: reviewers may run more complex test/CI scripts, thus, satisfying all the
 requirements above is just a preliminary, but not necessary sufficient step for
 getting the PR accepted as a valid candidate PR for the `master` branch.
 
+High quality commits help us review and merge you contributions, we attempt to
```

Comment:
In 370a294622233033fd262e90ee974786c89d1733:

This `,` should be a period, and capital `We`.

Reply:
Done, thanks.